lib/Cleantalk/Antispam/Integrations/KadenceBlocksAdvanced.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Cleantalk\Antispam\Integrations;
+
+class KadenceBlocksAdvanced extends IntegrationBase
+{
+    public function getDataForChecking($argument)
+    {
+        return ct_gfa(apply_filters('apbct__filter_post', $_POST));
+    }
+
+    public function doBlock($message)
+    {
+        $data = [
+            'redirect' => false,
+            'html' => '<div class="kadence-blocks-form-message kadence-blocks-form-error">' . $message . '</div>',
+            'headers_sent' => false,
+            'success' => true,
```

Comment:
Are you sure `'success' => true` should be not `false`?

Reply:
Furthermore, for ajax calls the better way is to use wp_send_json_error()/wp_send_json_success(). I believe we should use this function as code standard.